demisto_sdk/commands/common/errors.py

```diff
@@ -1562,6 +1572,8 @@
         "IN107",
         "DB100",
         "GR103",
+        # "IN158",
+        # "IN159",
```

Comment:
?

Reply:
Not sure if can be ignored or not. 
removed the commented out code,